runelite-client/src/main/java/net/runelite/client/plugins/itemstats/potions/StaminaPotion.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2016-2018, Adam <Adam@sigterm.info>
```

Comment:
Unless you're adam in disguise, you should change or append this with your own copyright.